invenio/modules/indexer/manage.py

```diff
@@ -0,0 +1,176 @@
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Manage indexer module."""
+
+from __future__ import print_function
+
+import sys
+
+from invenio.ext.script import Manager
+
+from .indexerext.config import ElasticSearchIndexFactory, NativeIndexFactory
+from .indexerext.importer.json_importer import \
+    JsonIndexerConfigurationImporter
+
+
+manager = Manager(usage=__doc__)
+
+option_yes_i_know = manager.option('--yes-i-know', action='store_true',
+                                   dest='yes_i_know', help='use with care!')
+option_quiet = manager.option('-q', '--quiet', action='store_true',
+                              dest='quiet', help='show less output')
+
+option_load_from_file = manager.option(
+    '-f', '--filename', dest="filename",
+    help='Specify a configuration\'s file (json format)')
+
+option_namespace = manager.option(
+    '-n', '--namespace', dest='namespace', action='append',
+    help='Specify the namespace (e.g. -n records or -n documents')
+
+option_fields = manager.option(
+    '-i', '--fields', dest='indexer_fields', action='append',
+    help='Specify the fields (e.g. -f title,author to choice title and '
+    'author from all namespaces or -f records.title to specify title of '
+    'namespace records')
+
+option_exclude_fields = manager.option(
+    '-e', '--exclude-fields', dest='exclude_fields', action='append',
+    help='Specify the excluded fields (e.g. -e title,author')
+
+option_range = manager.option(
+    '-r', '--range', dest='range', action='append',
+    help='Specify a range of ids')
+
+
+def load_config(filename=None):
+    """Load the configuration.
+
+    :param filename: file name
+    :return: raw configuration data
+    """
+    if filename:
+        with open(filename, 'r') as cfile:
+            data = cfile.read()
+    else:
+        data = ''
+        for line in sys.stdin:
+            data += line
+
+    return data
+
+
+def load_factory(indexer_name):
+    """Load factory.
+
+    :param indexer_name: indexer name (e.g. native or elasticsearch)
+    """
+    if indexer_name == 'elasticsearch':
+        return ElasticSearchIndexFactory()
+    else:
+        return NativeIndexFactory()
+
+
+def load_configuration(factory, data):
+    """Load configuration.
+
+    :param factory: index factory
+    :param data: configuration
+    :return: indexer configuration object
+    """
+    importer = JsonIndexerConfigurationImporter(
+        json_text=data,
+        factory=factory
+    )
+    return importer.load()
+
+
+@manager.command
+@option_namespace
+@option_fields
+@option_exclude_fields
+@option_load_from_file
+@option_yes_i_know
+def create(indexer, namespace=None, indexer_fields=None, exclude_fields=None,
+           filename=None, yes_i_know=False):
+    """Create indices."""
+    data = load_config(filename=filename)
+    factory = load_factory(indexer)
+    # config =
+    load_configuration(factory=factory, data=data)
```

Comment:
E.g. try to create elasticsearch indices from test_indices.json file:
```json
{
  "indices": {
    "index_title": {
      "description": "Test indexing title",
      "stemming_language": "it",
      "pre_index_actions": [
        "remove_stopwords"
      ],
      "field": "title",
      "native": {
        "tokenizer": "BibIndexAuthorCountTokenizer"
      }
    },
    "index_author": {
      "description": "Test indexing author",
      "stemming_language": "it",
      "pre_index_actions": [
        "remove_stopwords"
      ],
      "field": "title",
      "elasticsearch": {
        "analyzer": "myanalyzer"
      }
    }
  },
  "virtual_indices": {
    "virtual_index_all": {
      "description": "Test virtual indexing author and title",
      "indices": ["index_title", "index_author"],
      "namespace": "records"
    },
    "virtual_index_title": {
      "description": "Test virtual indexing title",
      "indices": ["index_title"],
      "namespace": "documents"
    },
    "index_author": {
      "description": "Test virtual indexing author",
      "indices": ["index_author"],
      "namespace": "records"
    }
  }
}
```

Then, from console:
`inveniomanage indexer create elasticsearch -f "test_indices.json"`
The `load_configuration()` functions return the `IndexerConfiguration` object.
(p.s. For now, I only implemented the `.json` importer, but we can think about also implement `.cfg.` importer and `.yml` importer... XD)

@marioskogias @jirikuncar